src/main/java/org/agbrothers/englishbot/processing/ProcessingCore.java

```diff
@@ -68,7 +68,7 @@ private void applyUserSate(String messageText, User user) {
         switch (messageText) {
             case LinkLabel.MAIN_MENU:
                 user.setStateId(State.MAIN_MENU);
-                lessonService.removeLesson(user.getChatId());
+                //lessonService.removeLesson(user.getLesson());
```

Comment:
why is this here?

Reply:
This is my inattention... Thanks for pointing this out.